lewis/scripts/run.py

```diff
@@ -182,10 +189,11 @@ def run_simulation(argument_list=None):  # noqa: C901
             print('\n'.join(simulation_factory.get_protocols(arguments.device)))
             return
 
-        protocol, options = parse_adapter_options(arguments.adapter_options)
+        protocols = parse_adapter_options(
+            arguments.adapter_options) if not arguments.no_interface else {}
```

Comment:
True, the line break makes this look a little weird.

I wasn't really sure about what is best in terms of behavior, so I tried this. But maybe it is indeed better to raise an error if both options are specified, it's a small change.